### PR TITLE
run 2 updates

### PR DESCRIPTION
**Pull request by nprouvost**

### CODE REVIEW

**Review by riga — CHANGES_REQUESTED** [review]
Nice work @aalvesan and @nprouvost !

I had a couple minor comments that I made inline below.

Besides that, I'd like to suggest to move all inference related changes to a second PR. The code is essentially a drop-in from the hbw analysis but I think we should discuss their approaches first and see if they apply to our analysis.

File: hbt/calibration/default.py [hunk 1]
```diff
@@ -52,3 +52,11 @@ def default(self: Calibrator, events: ak.Array, **kwargs) -> ak.Array:
...
 
     return events
+
+
+@calibrator(
+    uses=set(),
+    produces=set(),
+)
+def empty(self: Calibrator, events: ak.Array, **kwargs) -> ak.Array:
+    return events
```
[the remark this answers is not in the document]
Comment (nprouvost) [reply hbt/calibration/default.py]: removed

File: hbt/calibration/placeholder.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# coding: utf-8
```
[the remark this answers is not in the document]
Comment (nprouvost) [reply hbt/calibration/placeholder.py]: changed

File: hbt/config/variables.py [hunk 3]
```diff
@@ -164,3 +180,70 @@ def add_variables(config: od.Config) -> None:
...
+    config.add_variable(
+        name="hh_eta",
+        expression="hh.eta",
+        binning=(100, -3.0, 3.0),
+        unit="GeV",
+        x_title=r"$\eta$",
+    )
+
+    config.add_variable(
+        name="diTau_mass",
```
[the remark this answers is not in the document]
Comment (nprouvost) [reply hbt/config/variables.py]: changed

File: hbt/production/empty.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+from columnflow.production import Producer, producer
+from columnflow.production.normalization import normalization_weights
+from columnflow.production.categories import category_ids
+from columnflow.util import maybe_import
+# from hbt.production.btag import normalized_btag_weights
+# from columnflow.production.cms.electron import electron_weights
+# from columnflow.production.cms.muon import muon_weights
+# from hbt.production.features import features
+# from hbt.production.weights import normalized_pu_weight, normalized_pdf_weight, normalized_murmuf_weight
+# from hbt.production.tau import tau_weights, trigger_weights
```
[the remark this answers is not in the document]
Comment (nprouvost) [reply hbt/production/empty.py]: removed

File: hbt/production/empty.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
```
[the remark this answers is not in the document]
Comment (nprouvost) [reply hbt/production/empty.py]: changed

File: hbt/selection/empty.py [hunk 6]
```diff
@@ -0,0 +1,112 @@
+# coding: utf-8
```
[the remark this answers is not in the document]
Comment (nprouvost) [reply hbt/selection/empty.py]: removed

File: hbt/production/hh_mass.py [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+        **kwargs,
+    )
+
+    # mc-only weights
+    if self.dataset_inst.is_mc:
+        # normalization weights
+        events = self[normalization_weights](events, **kwargs)
+
+        # btag weights
+        # events = self[normalized_btag_weights](events, **kwargs)
```
[the remark this answers is not in the document]
Comment (nprouvost) [reply hbt/production/hh_mass.py]: Solved by using option 2: removing these weights from this producer, both producers should be used at once

File: hbt/selection/trigger.py [hunk 8]
```diff
@@ -45,7 +45,15 @@ def trigger_selection(
...
-        fired = events.HLT[trigger.hlt_field] == 1
+
+        try:
+            fired = events.HLT[trigger.hlt_field] == 1
+        except:
+            import traceback
+            traceback.print_exc()
+            from IPython import embed
+            embed(header="trigger selection")
+
```
[the remark this answers is not in the document]
Comment (nprouvost) [reply hbt/selection/trigger.py]: removed

**Review by riga — APPROVED** [review]

**Pull request merged**